app/views/candidate_interface/personal_details/languages/edit.html.erb

```diff
@@ -0,0 +1,26 @@
+<% content_for :title, title_with_error_prefix(t('page_titles.languages'), @languages_form.errors.any?) %>
+<% content_for :before_content, govuk_back_link_to(candidate_interface_application_form_path, 'Back to application') %>
```

Comment:
Should this go back to the nationality page rather than the main application page? (the nationality page goes back to the personal details page).

Reply:
Good shout. Have done.